File: quests/freportw/Joshua.pl
############################################
# ZONE: West Freeport (freportw)
# DATABASE: PEQ-Velious
# LAST EDIT DATE: May 1,2005
# VERSION: 1.0
# DEVELOPER: MWMDRAGON
#
# *** NPC INFORMATION ***
#
# NAME: Joshua
# ID: 9121
# TYPE: Warrior
# RACE: Human
# LEVEL: 5
#
# *** ITEMS GIVEN OR TAKEN ***
#
# bucket of water ID-29008
# bucket of pure water ID-29009
#
# *** QUESTS INVOLVED IN ***
#
#
#
# *** QUESTS AVAILABLE TO ***
#
#
#
# *** NPC NOTES ***
#
#
#
############################################

sub EVENT_SAY
{
if($text=~/Hail/i)
 {
 quest::say("Welcome to freeport $name.");
 }


}

sub EVENT_ITEM
{
   # bucket of water ID-29008
   if($itemcount{29008} == 1)
   {
   quest::me("Joshua dips his emaciated hand into the bucket and brings the water to his mouth. As he does, you hear a soft thump from outside and notice that through the window his sister has collapsed. A glimmer enfolds her body, and you notice the water in Joshua's hand gleam a pure, bright light for a moment. 'Thank you, sir. I hope my sister returns soon. I feel better for some reason.");

   quest::ding();
   # bucket of pure water ID-29009
   quest::summonitem("29009");
   }
}

#END of FILE Zone:freportw  ID:9121 -- Joshua

File: freeporteast/Joshua.lua
-- no response to hail
-- items: 29008, 29009

function event_trade(e)
	local item_lib = require("items");

	if(item_lib.check_turn_in(e.trade, {item1 = 29008})) then -- bucket of water
		e.self:Emote("dips his emaciated hand into the bucket and brings the water to his mouth. As he does, you hear a soft thump from outside and notice that through the window his sister has collapsed. A glimmer enfolds her body, and you notice the water in Joshua's hand gleam a pure, bright light for a moment. 'Thank you, sir. I hope my sister returns soon. I feel better for some reason.'");
		e.other:SummonItem(29009); -- bucket of pure water
		e.other:Ding();
		e.other:AddEXP(500);
	end
	item_lib.return_items(e.self, e.other, e.trade)
end

-- END of FILE Zone:freportw -- Joshua